benchmark/csharp/Tpch/VectorDataFrameFunctions.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Data.Analysis;
+
+namespace Tpch
+{
+    internal static class VectorDataFrameFunctions
+    {
+        internal static PrimitiveDataFrameColumn<double> ComputeTotal(PrimitiveDataFrameColumn<double> price, PrimitiveDataFrameColumn<double> discount, PrimitiveDataFrameColumn<double> tax)
+        {
+            if ((price.Length != discount.Length) || (price.Length != tax.Length))
+            {
+                throw new ArgumentException("Arrays need to be the same length");
+            }
+
+            return (PrimitiveDataFrameColumn<double>)(price * (1 - discount) * (1 + tax));
```

Comment:
This looks so much better than the Arrow implementation.

When we get strongly typed operators, we can cut out the cast and it will look even better.

Reply:
Yea this looks really good. Also, it should be as performant right? (We will run a separate benchmark as well)

Reply:
The Microsoft.Data.Analysis library doesn't use SIMD operations yet. But eventually the goal is to use them in these math operations.

For now, my assumption would be this would be on the same order as looping over the rows and doing the calculations 1-by-one. Getting a measurement on this would be great.